src/sentry/api/endpoints/organization_projects.py

```diff
@@ -106,6 +106,12 @@ def get(self, request, organization):
                     queryset = queryset.filter(Q(name__icontains=value) | Q(slug__icontains=value))
                 elif key == 'id':
                     queryset = queryset.filter(id__in=value)
+                elif key == "team":
+                    team_list = list(Team.objects.filter(slug__in=value))
+                    queryset = queryset.filter(teams__in=team_list)
+                elif key == "-team":
```

Comment:
Could we make this `!team` to fit better with our other search syntax?

Reply:
Sure, updated